tests/test_grpc.py

```diff
@@ -89,14 +89,15 @@ def test_participant_rendezvous_later(participant_stub):
     """
 
     # populate participants
-    coordinator = Coordinator(minimum_participants_in_round=10, fraction_of_participants=1.0)
+    coordinator = Coordinator(
+        store=FakeS3Store(), minimum_participants_in_round=10, fraction_of_participants=1.0
```

Comment:
here you use `FakeS3Store` but is it necessary? also, in some other tests e.g. `test_monitor_heartbeats` you don't specify the store...

Reply:
It's not necessary indeed, I removed it. The default store is `DummyStore` which is enough for most tests, so we don't need to specify it.